Source/HelixToolkit.Wpf.SharpDX/Controls/RenderContext.cs

```diff
@@ -35,6 +35,8 @@ public class RenderContext : IDisposable
 
         public Matrix ProjectrionMatrix { get { return this.projectionMatrix; } }
 
+        public Matrix WorldMatrix { get { return worldMatrix;} }

```

Comment:
This property is created so that third parties which create their own types which inherit from `GeometryModel3D` can overload the `Render` method.